### PR TITLE
[WIP] Chapter 2 - lighting

### PR DESCRIPTION
**Pull request by frederikja163**

### CODE REVIEW

File: Chapter 2/1 - Colors/Window.cs [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+namespace LearnOpenTK
+{
+    //In this chapter we will focus on how to use lighting to make our games and other applications more lifelike
+
+    //In this first part the focus will mainly be on setting up a scene for testing the different coloring options.
+    //We draw two cubes one at 0,0,0 for testing our light shader, the second one is drawn where we have the light.
+    //Furthermore in the shaders we have set up some basic physically based coloring.
+    public class Window : GameWindow
+    {
+        //The vertices are now used to draw cubes, notice how we dont use the texture coords anymore,
```
Comment (devvoid) [inline Chapter 2/1 - Colors/Window.cs]: Rephrase a little bit. Perhaps something more along the lines of "For this example, we aren't using texture coordinates. You can use textures with lighting, but for simplicity's sake, we'll just use a solid color here."

File: Chapter 2/2 - Basic lighting/Window.cs [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+using OpenTK;
+using OpenTK.Graphics;
+using OpenTK.Graphics.OpenGL4;
+using OpenTK.Input;
+using LearnOpenTK.Common;
+
+namespace LearnOpenTK
+{
+    //In this tutorial we set up some basic lighting and look at how the phong model works
+    //For more insigt into how it all works look at the web version. If you are just here for the source,
```
Comment (devvoid) [inline Chapter 2/2 - Basic lighting/Window.cs]: Typo.

File: Chapter 2/2 - Basic lighting/Shaders/lighting.frag [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+uniform vec3 lightColor;
+uniform vec3 lightPos;
+uniform vec3 viewPos;
+
+in vec3 Normal;
+in vec3 FragPos;
+
+void main()
+{
+    float ambientStrength = 0.1;
```
Comment (devvoid) [inline Chapter 2/2 - Basic lighting/Shaders/lighting.frag]: Most of the fragment shaders still need comments explaining what's going on.

File: Chapter 2/2 - Basic lighting/Window.cs (hunk 2 above)
Comment (frederikja163) [reply Chapter 2/2 - Basic lighting/Window.cs]: Fixed in next comit

File: Chapter 2/1 - Colors/Window.cs (hunk 1 above)
Comment (frederikja163) [reply Chapter 2/1 - Colors/Window.cs]: Fixed in next commit

**Comment by devvoid** [comment]
Looks good to me. Ready for merge, if @Perksey doesn't have any issues.

**Comment by Perksey** [comment]
Fine by me, I'm more focused on the web versions being solid.

**Pull request merged**